src/index.js

```diff
@@ -0,0 +1,68 @@
+// eslint-disable-next-line no-unused-vars
+import css from './app.css';
+
+const todo = [
+  {
+    description: 'wash the dishes',
+    completed: true,
+    index: 5,
+  },
+  {
+    description: 'complete To Do list project',
+    completed: false,
+    index: 1,
+  },
+];
+
+const loadTodo = (parent) => {
+  todo.sort((a, b) => a.index - b.index);
+
+  todo.forEach((task) => {
+    const el = document.createElement('li');
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.id = `task-${task.index}`;
+    // eslint-disable-next-line no-use-before-define
```

Comment:
- Please, let's try to write codes that conform to the eslint as it helps us write readable and maintainable code. Please, remove and fix the linter issue